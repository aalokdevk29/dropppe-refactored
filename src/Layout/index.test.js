import { configure, shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
// import renderer from 'react-test-renderer';
import TestRenderer from 'react-test-renderer';
import React from 'react';
import {HeaderConatiner} from './index'

configure({ adapter: new Adapter() });

describe('HeaderConatiner component', () => {
  it('should render properly', () => {
    const tree = TestRenderer.create(<HeaderConatiner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
})

it('should render a div class', () => {
  const wrapper = shallow(<HeaderConatiner />);
  expect(wrapper.find('.header').exists()).toBe(true);
});
