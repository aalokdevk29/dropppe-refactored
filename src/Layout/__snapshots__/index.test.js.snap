// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderConatiner component should render properly 1`] = `
Array [
  <div
    className="header"
  >
    <div
      className="container headerImageWrapper"
    >
      <img
        alt="logo"
        className="headerImage"
        src="./assets/droppe-logo.png"
      />
    </div>
  </div>,
  <span
    className="container main innerFragment"
  >
    <img
      alt="image1"
      className="showImg"
      src="./assets/img1.png"
    />
    <img
      alt="image2"
      className="showImg"
      src="./assets/img2.png"
    />
  </span>,
]
`;
